### PR TITLE
New charts lib for Benefits Lines graph, #956. Feedback welcome

### PR DESCRIPTION
**Pull request by knod**
Building it up.

### CODE REVIEW

**Comment by knod** [comment]
Not a WIP anymore! Ready for review!

File: package.json [hunk 1]
```diff
@@ -5,12 +5,14 @@
   "dependencies": {
     "ajv": "6.5.4",
     "chart.js": "2.7.2",
+    "highcharts": "^6.2.0",
```
Comment (knod) [inline package.json:8]: Not sure whether to mess with this caret or not. This is what the react-jsx-highcharts instructions said to install.

File: package.json [hunk 2]
```diff
@@ -5,12 +5,14 @@
...
     "lodash": "4.17.10",
     "moment": "2.22.1",
     "react": "16.3.2",
     "react-chartjs-2": "2.7.2",
     "react-dom": "16.3.2",
     "react-helmet": "5.2.0",
+    "react-jsx-highcharts": "^3.3.1-alpha.1",
```
Comment (knod) [inline package.json:15]: Not sure why `npm install...` pulled in an alpha version. Not sure what to do about that either.

**Comment by knod** [comment]
Also helps with #546. Doesn't actually fix it, dangit. Just makes it more likely to be less of a problem. Handling the display based on the width of an actual title is a much trickier proposition.

File: src/index.css [hunk 3]
```diff
@@ -240,15 +240,49 @@ input[type=number]::-webkit-outer-spin-button {
 .graph-holder {
   position: relative;
   min-height: 7rem;
-  display: flex;
-  justify-content: flex-end;
-  align-items: flex-end;
+  /*display: flex;*/
```
Comment (turnerhayes) [inline src/index.css]: Can you remove these lines rather than comment them out?

File: src/test/forms/output/GraphHolder.test.js [hunk 4]
```diff
@@ -9,7 +9,7 @@ const passedActivePrograms = (wrapper) => {
   return wrapper.find('Graph').prop('activePrograms');
 };
 
-describe('<GraphHolder>', () => {
+describe.skip('<GraphHolder>', () => {
```
Comment (turnerhayes) [inline src/test/forms/output/GraphHolder.test.js:14]: Is this supposed to be skipped? If so, could you add a comment for why it's being skipped?

**Review by turnerhayes — APPROVED** [review]
Nothing here that blocks the PR; mostly formatting/comments. It would be great if you wanted to address them here, but if not we can have a separate issue to address them.

File: src/test/forms/output/GraphHolder.test.js (hunk 4 above)
Comment (knod) [reply src/test/forms/output/GraphHolder.test.js:14]: Great idea! Done!

File: src/index.css (hunk 3 above)
Comment (knod) [reply src/index.css]: Thanks!

File: package.json (hunk 1 above)
Comment (knod) [reply package.json:8]: @todo

File: package.json (hunk 2 above)
Comment (knod) [reply package.json:15]: @todo

**Pull request merged**